presentation/src/main/java/org/sagebionetworks/assessmentmodel/presentation/compose/SageButtons.kt

```diff
@@ -117,8 +118,8 @@ fun BlackNextButton(
     enabled: Boolean = true) {
     BlackButton(onClick = onClick,
         iconDrawable = R.drawable.ic_right_arrow,
-        enabled = enabled,
-        modifier = Modifier.width(56.dp))
+        text = stringResource(R.string.next),
```

Comment:
So... what happens if this string isn't in the app resources?

Reply:
This is enforced at compile time.